bell.py

```diff
@@ -1,16 +1,26 @@
+from socket import timeout
+import urllib.request
+import subprocess
+import argparse
+import json
+import sys
+import os
+
 from loguru import logger
```

Comment:
Could lose this dependency with something like:

```
from logging import getLogger
logger = getLogger(<desc>)
```

Reply:
I was about to do that next 👍